stable/traefik/README.md

```diff
@@ -106,7 +106,7 @@ The following tables lists the configurable parameters of the Traefik chart and
 | `ssl.defaultCert`               | Base64 encoded default certificate                                    | A self-signed certificate                 |
 | `ssl.defaultKey`                | Base64 encoded private key for the certificate above                 | The private key for the certificate above |
 | `acme.enabled`                  | Whether to use Let's Encrypt to obtain certificates                  | `false`                                   |
-| `acme.challengeType`            | Type of ACME challenge to perform domain validation. `tls-sni-01` or `dns-01` | `tls-sni-01`                     |
+| `acme.challengeType`            | Type of ACME challenge to perform domain validation. `tls-sni-01`, `http-01` or `dns-01` | `tls-sni-01`                     |
```

Comment:
There's a corresponding comment in `values.yaml` that should also be updated. I have the sense that a lot of people bypass the documentation and go straight to something like `helm inspect values stable/traefik` to explore available options, so having comments in `values.yaml` is useful for that crowd.

Reply:
You're right, totally forgot those. Fixed and squashed into commit.